pkg/server/image_pull.go

```diff
@@ -17,14 +17,226 @@ limitations under the License.
 package server
 
 import (
-	"errors"
+	"encoding/json"
+	"fmt"
+	"time"
 
+	"github.com/containerd/containerd/content"
+	containerd_images "github.com/containerd/containerd/images"
```

Comment:
s/containerd_images/containerdimages

We don't use understore in imports in k8s.